src/interop/js-interop/js-types.md

```diff
@@ -202,14 +199,17 @@ avoid. See issue [#4841] for more details.
 JS has both a `null` and an `undefined` value. This is in contrast with Dart,
 which only has `null`. In order to make JS values more ergonomic to use, if an
 interop member were to return either JS `null` or `undefined`, the compiler maps
-these values to Dart `null`. Therefore a member like `value` in the following example
-can be interpreted as returning a JS object, JS `null`, or `undefined`:
+these values to Dart `null`. Therefore a member like `value` in the following
+example can be interpreted as returning a JS object, JS `null`, or `undefined`:
 
 ```dart
 @JS()
 external JSObject? get value;
 ```
 
+If the return type was not declared as nullable, then the compiler will throw an
```

Comment:
nit: s/compiler/program? (technically the compiler will complete normally, but the output program will throw :smile: )

Reply:
Yes, thanks :)